gcp_variant_transforms/pipeline_common.py

```diff
@@ -66,82 +66,68 @@ def parse_args(argv, command_line_options):
   for transform_options in options:
     transform_options.validate(known_args)
   _raise_error_on_invalid_flags(pipeline_args)
+  known_args.input_patterns = _get_input_patterns(
+      known_args.input_pattern, known_args.input_file)
   return known_args, pipeline_args
 
+def _get_input_patterns(input_pattern, input_file):
+  return (
+      _get_file_names(input_file) if input_pattern is None else [input_pattern])
+
+def _get_file_names(input_file):
+  # type (str) -> List(str)
+  """ Reads all input file and extracts list of patterns out of it."""
+  result = []
+  if not filesystems.FileSystems.exists(input_file):
+    raise ValueError('Input file {} doesn''t exist'.format(input_file))
+  with filesystems.FileSystems.open(input_file) as f:
+    for _, l in enumerate(f):
+      result.append(l)
+    if not result:
+      raise ValueError('Input file {} is empty.'.format(input_file))
+    return result
 
 def get_pipeline_mode(
-    input_pattern,
-    input_file,
+    input_patterns,
+    input_file=None,
     optimize_for_large_inputs=False):
   # type: (str, bool) -> int
   """Returns the mode the pipeline should operate in based on input size."""
   if optimize_for_large_inputs:
     return PipelineModes.LARGE
 
-  if input_pattern is not None:
-    total_files = _get_pipeline_mode_for_pattern_input(input_pattern)
-  else:
-    total_files = _get_pipeline_mode_for_file_input(input_file)
+  match_results = filesystems.FileSystems.match(input_patterns)
+  if not match_results:
```

Comment:
These validations should already be handled.

Reply:
So, as per offline convo, I tried to follow the older conventions and expand existing validations. Removed now.